src/debugger/adapter/DebugAdapter.js

```diff
@@ -36,7 +43,9 @@ class PrepackDebugSession extends DebugSession {
     this.setDebuggerColumnsStartAt1(true);
   }
   _clientID: void | string;
-  _adapterChannel: AdapterChannel;
+  _adapterChannel: void | AdapterChannel;
+  // Buffering requests that are sent before Prepack channel has been established.
+  _bufferedBreakpointRequests: Array<PrematureBP> = [];
```

Comment:
Per our discussion offline, once we delay InitializedEvent response to launch, _bufferedBreakpointRequests should not be necessary. We do have runtime check to assert this assumption/protocol at runtime.